circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/DropTableService.java

```diff
@@ -56,13 +100,52 @@ public void removeTableParamsAndDrop(
       }
       client.alter_table(databaseName, tableName, table);
     }
-    LOG
-        .info("Dropping table '{}.{}'.", table.getDbName(), table.getTableName());
-    client.dropTable(table.getDbName(), table.getTableName(), false, true);
+    LOG.info("Dropping table '{}.{}'.", databaseName, tableName);
+    client.dropTable(databaseName, tableName, false, true);
+  }
+
+  private void deleteData(DataManipulationClient dataManipulationClient, String replicaDataLocation) {
+    try {
+      LOG.info("Dropping table data from location: {}.", replicaDataLocation);
+      boolean dataDeleted = dataManipulationClient.delete(replicaDataLocation);
+      LOG.info("Data deleted: {}.", dataDeleted);
+    } catch (IOException e) {
+      LOG.info("Could not drop replica table data at location:{}.", replicaDataLocation);
+    }
+  }
+
+  private void deletePartitionData(
+      DataManipulationClient dataManipulationClient,
+      String replicaTableLocation,
+      List<String> replicaPartitionLocations) {
+    try {
+      LOG.info("Dropping partition data from base location: {}.", replicaTableLocation);
```

Comment:
```suggestion
      LOG.info("Deleting partition data from base location: {}.", replicaTableLocation);
```
(although, I'm not sure it's even worth logging or using this as theoretically the partition locations could be completely unrelated to the table's base location.

Reply:
Yes this is not useful to log and might even be confusing.